gateway-ha/src/main/java/io/trino/gateway/ha/handler/ProxyUtils.java

```diff
@@ -52,6 +53,8 @@ public final class ProxyUtils
      * capitalization.
      */
     private static final Pattern QUERY_ID_PARAM_PATTERN = Pattern.compile(".*(?:%2F|(?i)query_?id(?-i)=|^)(\\d+_\\d+_\\d+_\\w+).*");
+    public static final ImmutableSet<String> QUERY_STATE_PATH = ImmutableSet.of("queued", "scheduled", "executing");
```

Comment:
Use an interface for declaration:
```suggestion
    public static final Set<String> QUERY_STATE_PATH = ImmutableSet.of("queued", "scheduled", "executing");
```

Reply:
Done